src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -47,9 +47,13 @@ import { DatePickerTypes } from "./DatePickerTypes";
       dateType: DatePickerTypes.Full,
       dateFormat: "yyyy-MM-dd",
       dateRange: false,
+      initialDate: "1/1/2021",
```

Comment:
It would be nice to have `initialDate` and `initialDateTo` listed consecutively in the story ArgsTable.